notebooks/Photon Sanity Check.scala

```diff
@@ -0,0 +1,25 @@
+// Databricks notebook source
```

Comment:
Label this notebook with a number, or put it in libs; still needs to be numbered though.